FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -236,16 +226,9 @@ - (void)realtimeFetchConfigWithNoExpirationDuration:(NSInteger)fetchAttemptNumbe
       } else {
         FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000053",
                     @"A fetch is already in progress. Ignoring duplicate request.");
-        NSError *error =
-            [NSError errorWithDomain:FIRRemoteConfigErrorDomain
-                                code:sFIRErrorCodeConfigFailed
-                            userInfo:@{
-                              NSLocalizedDescriptionKey :
-                                  @"FetchError: Duplicate request while the previous one is pending"
-                            }];
         return [strongSelf reportCompletionWithStatus:FIRRemoteConfigFetchStatusFailure
```

Comment:
yeah so this would just be a no-op action. There would be a log in the developer's console but no error would make it's way to the listener or out of RCNConfigFetch

Reply:
but then the listener would "miss" this update right? and it wouldn't be triggered until the next time a realtime fetch was successful?

Reply:
In this case the Fetch would be triggered by the user anyway and it would follow whatever activation steps had been set up for the regular Fetch